cypress/e2e/RoundTrips/SimpleProject.feature

```diff
@@ -0,0 +1,473 @@
+    And   I set context field 'secondModelFolder' with 'model2'
+    And   I set context field 'secondModelName' with 'secondModelTest'
+    And   I set context field 'secondModelFile' with 'model2/main.tf'
+    And   I set context field 'thirdModelName' with 'thirdModelTest3Githubator.yml'
+    And   I set context field 'modelRenamed' with 'newModelTest'
+    And   I visit the '/'
+
+    ## 100 Home page should display an empty project message and one template project card
+    Then I expect '[data-cy="project-grid-empty"]' exists
+    And  I expect '[data-cy="project-grid-empty"]' is 'No projects, please create a new project to have one here 😉'
+    And  I expect '[data-cy="template-card_project_template"]' exists
+    And  I expect '[data-cy="template-card_project_template"] [data-cy="title-container"]' is 'Project template'
+
+    ## 101 Creating a project with an empty name should display an error
+    When I click on '[data-cy="create-project-button"]'
+    And  I click on '[data-cy="create-project-form"] [data-cy="submit-button"]'
+    Then I expect '[data-cy="create-project-form"] [role="alert"]' is 'Please type something'
+
+    And I visit the '/'
+    And I expect '[data-cy="create-project-button"]' exists
+
+    ## 102 Create intial project
+    When I click on '[data-cy="create-project-button"]'
+    And  I set on '[data-cy="create-project-form"] [data-cy="name-input"]' text '{{projectName}}'
+    And  I click on '[data-cy="create-project-form"] [data-cy="submit-button"]'
+
+    ## 103  Created project should redirect to models page and send positive toast
+    Then I expect 'positive' toast to appear with text 'Project has been created 🥳!'
+    And  I expect current url is '{{projectName}}/models'
+
+    When I visit the '/'
+
+    ## 104 Created project should be in the projects list
+    Then I expect '[data-cy="project-card_{{projectName}}"]' appear 1 time on screen
+    And  I expect '[data-cy="project-card_{{projectName}}"] [data-cy="title-container"]' is '{{projectName}}'
+
+    ## 105 Created project should be in the left drawer
+    And I expect '[data-cy="project-expansion-item"] [data-cy="item_{{projectName}}"]' exists
+    And I expect '[data-cy="project-expansion-item"] [data-cy="item_{{projectName}}"]' is '{{projectName}}'
+
+    When I visit the '/'
+    Then I expect '[data-cy="create-project-button"]' exists
+
+    ## 106 Create project with an already existing project name should display an error
+    And  I click on '[data-cy="create-project-button"]'
+    And  I set on '[data-cy="create-project-form"] [data-cy="name-input"]' text '{{projectName}}'
+    And  I click on '[data-cy="create-project-form"] [data-cy="submit-button"]'
+    And  I expect '[data-cy="create-project-form"] [role="alert"]' is 'Project name already exists.'
+
+    When I visit the '/'
+    Then I expect '[data-cy="project-card_localProjectTest"] [data-cy="rename-button"]' exists
```

Comment:
why not using `project-card_{{projectName}}` ? (it took me a minute to remember where `localProjectTest` comes from)

Reply:
done